coordinator/cluster/util.go

```diff
@@ -0,0 +1,62 @@
+package cluster
+
+import (
+	"errors"
+	"sort"
+
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/agent"
+)
+
+type sortableAgent struct {
+	order int
```

Comment:
From what I understand this is a reference to the position/index in the original agents array.  
If so, I think it might make more sense to call it `index`. Thoughts?

Reply:
index at the beginning but it slightly evolves to an order actually :) I think both names makes sense.

Reply:
Interesting. My understanding was that the order of the final array depends only on the value of `users`.